src/kibana/plugins/visualize/editor/sidebar.html

```diff
@@ -1,7 +1,6 @@
 <div class="sidebar-container">
-  <form class="sidebar-list"
-        ng-class="{'hide-errors': hideErrors}"
-        ng-submit="visualizeEditor.$invalid ? dontApply() : stageEditableVis()"
+  <form class="sidebar-list in-time-errors"
+        ng-submit="stageEditableVis()"
```

Comment:
That's only if the validation is based on html5 validation. things like query validation are not html5 based which is why we check for `$invalid`